packages/cozy-sharing/src/state.spec.js

```diff
@@ -487,6 +489,16 @@ describe('getSharedDocIdsBySharings method', () => {
   })
 })
 
+describe('getPermissionDocIds method', () => {
+  it('should return the permission by docsid', () => {
+    const docs = getPermissionDocIds(PERM_2)
+    expect(docs).toEqual(['folder_2'])
+  })
+  it('should return nothing if there is no document', () => {
+    const docs = getPermissionDocIds(PERM_WIHTOUT_DOC)
+    expect(docs).toEqual([])
```

Comment:
It seems a bit surprising here to return an empty list when the permission is on the whole doctype. What happens if the permission has really no documents. Does it switch to global permission ? I think returning null here would be clearer.

Reply:
Let me create the issue to discuss this thing because it's not clear to me how to correctly deal with that issue yet. For now, let's just fix the crash. (I edited the PR's description)